eMarket/app/models.py

```diff
@@ -24,8 +24,13 @@ def __str__(self):
 class Cart(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE, limit_choices_to={'user_type': 'buyer'})
     product = models.ForeignKey(Product, on_delete=models.CASCADE)
-    quantity = models.PositiveIntegerField(default=1)
-
-
+    quantity = models.PositiveIntegerField(default=0)
+    def get_total(self):
+        return self.quantity * self.product.price
+    
 class Order(models.Model):
-    items = models.ForeignKey(Cart, on_delete=models.CASCADE)
\ No newline at end of file
+    user = models.ForeignKey(User, on_delete=models.CASCADE, limit_choices_to={'user_type': 'buyer'})
+    cart = models.ManyToManyField(Cart)
+    quantity = models.PositiveIntegerField(default=1)
```

Comment:
This default and the one on line 27 don't line up is this intended

Reply:
I'm referring to quantity here

Reply:
Yeah there will be no order created from an empty cart (nonexistent cart) so the order quantity should at least start at 1 imo. But also I haven't tested that to account for errors so I'll double check how it handles that